client/app/components/navbar/navbar.service.js

```diff
@@ -0,0 +1,19 @@
+{
+  class Navbar {
+    static buildMenuHeader(text) {
+      return `
```

Comment:
Why do you build html into js service?

Reply:
i don't understand either, if you really want to do that, using $compile et importing the template would looks better